lib/errors.js

```diff
@@ -71,6 +71,8 @@ ValidationError.serializers = {
     `Invalid value of a enum in a property '${error.property}' ` +
     `allowed: ${wrap(error.info.expected)}, actual: '${error.info.actual}'`,
   undefinedEntity: error => `Undefined ${error.info} '${error.property}'`,
+  invalidAddProp: error =>
+    `Property '${error.property}' can't be found in validation schema.`,
```

Comment:
Please change error message template so that reported message is similar to:
`'SchemaName.Property' must be a valid instance of one of domain 'A', category 'B', ...`.